### DISCUSSION TITLE
Issues with Opening of .ipynb files in Github repositories.

### DISCUSSION THREAD

**Discussion by PavanCyborg** [community/community#121505] [Repositories]
### Select Topic Area

Product Feedback

### Body

Hello!!!.

I am facing the issue while opening **.ipynb** files in any of the github repositories. I again need to clone the repository and open it locally with my jupyter notebook to view the file. I am unable to load the file in the github portal.

![image](https://github.com/community/community/assets/66762893/17600f8d-8a2a-431d-9080-599ba6417a39)

The above image consists of error what i am facing when i try to open any **.ipynb** file.

Please resolve this issues as soon as possible. Thanks in Advance !!!!

**Comment by yoannchaudet** [comment]
Removing the "Pages" label as this is not a GitHub Pages issue.

**Reply by PavanCyborg** [reply]
Oh okay!!!. Actually i am new to Github so i don't knew much about it. So, do you have any update over the issue?

**Reply by LiteBrite82** [reply]
Hi @PavanCyborg !

Welcome to GitHub Community Discussions!!!! 

@yoannchaudet is right; the `Pages` category is for discussions concerning the `GH Pages` product. I think your question would get the best community support in the `Code Search and Navigation` category. 

I'll move it there for you 😄 

Are you able to share the repo link to see if others can replicate the issue? I did a random search for .ipynb files in GitHub and was able to view them on my machine. [example](https://github.com/julienr/ipynb_playground/blob/master/bitcoin/dumbcoin/dumbcoin.ipynb)

**Reply by look** [reply]
@LiteBrite82 I moved this to `Repositories` because they are responsible for file rendering.

**Reply by PavanCyborg** [reply]
Thanks @look and @LiteBrite82 for your concern.

I have shared the repository link to all my team members. They are also facing the same issue. In some devices after multiple refreshes we are able to view the **.ipynb** file. But most of the time it doesn't work.
